client/app/src/js/controllers/screenshots.js

```diff
@@ -4,9 +4,12 @@
  * Handles display of a user screenshots.
  */
 
-function ScreenshotsController($location, screenshots, Screenshot) {
+function ScreenshotsController($stateParams, $location, screenshots, Screenshot, User) {
+
   this.screenshots = screenshots;
   this.url = '';
+  this.isUser = User.getUser() !== null && User.getUser().username === $stateParams.username;
+
   this.addScreenshot = function(url){
     Screenshot.addScreenshot(url)
       .success(function(data){
```

Comment:
I can't comment on line 19, but same question about `this.getScreenshots` redundant when we're `resolve`'ing it?